src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--sun calendar__month-length--29">
+      <div class="calendar__ceil calendar__ceil--1 calendar__ceil--day--thu"></div>
+      <div class="calendar__ceil calendar__ceil--2"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
But if I will set class "calendar__day" for each day, how I can put numbers via eachloop in scss? Or I should replace word "ceil" to "day"?

Reply:
Okay, I get it